styleguide.css

```diff
@@ -364,22 +423,135 @@ p a {
 
 p a:after {
     position: absolute;
-    top:1rem;
+    top: 1rem;
     bottom: 0.5rem;
     right: 0;
     left: 0;
     display: block;
     background-color: #bde513;
     overflow: hidden;
-    content: "";
+    content: '';
     transform: translateY(35%);
     transition: all .3s;
-    z-index:-1;
+    z-index: -1;
 }
 
 p a:hover:after {
-    top:0;
+    top: 0;
     bottom: 0.5rem;
     transform: translateY(0);
     transition: all .3s;
 }
+
+/* Menu */
+
+button.menu-button {
+    position: fixed;
+    display: block;
+    top: 1rem;
+    left: 1rem;
+    width: 9.5rem;
+    height: 9.5rem;
+    border: none;
+    background: transparent;
+    padding: 0;
+    margin: 0;
+    cursor: pointer;
+    z-index: 2;
+}
+
+button.menu-button:focus {
+    outline: none;
+}
+
+button.menu-button:before,
+button.menu-button:after {
+    content: '';
+    height: 0.3rem;
+    width: 3.5rem;
+    margin: 0;
+    left: 3rem;
+    background: #00283C;
+    position: absolute;
+    display: block;
+    transition: all .3s ease-in-out;
+}
+
+button.menu-button:before {
+    top: 4rem;
+}
+
+button.menu-button:after {
+    bottom: 4rem;
+}
+
+button.menu-button:after.active {
+    top: 0;
+}
+
+button.menu-button.active:before {
+    top: 4.6rem;
+    transform: rotate(45deg);
+}
+button.menu-button.active:after {
```

Comment:
how is this different from line 488 `button.menu-button:after.active`?

Reply:
`button.menu-button:after.active` line 488 styles the behavior of the menu button, which is not the default button behavior